data/mainnet/tokens.js

```diff
@@ -7185,4 +7185,23 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DAI', 'WETH'],
   },
+  convex_cvxCRV: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: VAULT_CATEGORIES_IDS.INACTIVE,
+    displayName: 'Convex CRV',
+    apyIconUrls: [],
```

Comment:
We'll need the icons and token symbols for the different reward tokens in here. CRV, CVX and 3CRV in this case